### DISCUSSION TITLE
Questions regarding DIA-NN quantification

### DISCUSSION THREAD

**Discussion by hguturu** [vdemichev/DiaNN#318] [Q&A] [answered]
If DIA-NN `.quant` files are created on a bunch of samples separately and then reprocessed in a big group run, is normalization and MaxLFQ redone? 

Or do the original `.quant` files need to be processed with ` --no-norm` and ` --no-maxlfq` and then when processed in a big group run these flags need to be turned on? 

What is the difference between `--no-norm` and `--no-maxlfq`? I assume `--no-norm` refers to the `RT-dependent normalisation` from the FAQs, but can you describe what the normalization is? Is it some kind of median normalization based on RT-bins?

**Comment by vdemichev** [comment, marked as answer]
Normalisation (on the precursor level) and protein quant (via MaxLFQ or otherwise) are only applied right before the report generation, that is after summarising all the info from .quant files. That is during the generation of .quant files these algorithms are not used. There's no connection between normalisation and MaxLFQ algorithms.

**Reply by vdemichev** [reply]
Yes, precursor normalisation is first applied as described in the DIA-NN paper, and then, if selected in the GUI, RT-dependent normalisation can be applied, indeed like a median using RT-bins

**Reply by hguturu** [reply]
Got it. Thanks for clarifying the the lack of connection between precursor normalization and protien MaxLFQ.

Regarding precursor normalization, the DIA-NN paper discusses a cross-run normalisation based on the total signal. Does `--no-norm` turn off this normalization or the `RT-dependent normalization` or both? 

I assume `--global-norm` allows you to turn off `RT-dependent normalization`, but leave the normalization as in the paper. Can you do the converse? Turn off the normalization as in the paper, but leave the `RT-dependent normalization`?

**Reply by vdemichev** [reply]
--no-norm completely turns off all normalisation steps.
No, not really possible to use RT-dependent normalisation while having the basic global one turned off.

**Reply by hguturu** [reply]
Thank you! That makes sense.

**Comment by hguturu** [comment]
One followup regarding quantification - when using the `--relaxed-prot-inf` are the razor peptides used for quantification? If so is there a way to break up the quantification from razor vs non-razor peptides? Similar to how fragpipe does total/unique/razor intensity. Or at least compute with unique only or total only?

When not using default protein inference, I assume razor peptides are not an issue since they will be placed in a separate protein group?

**Reply by vdemichev** [reply]
Each peptide in DIA-NN output is matched to a protein group. So each protein groups has a list of peptides matched to it. Only these are used to quantify this protein group. If you'd like only proteotypic peptides used for quant, then Genes.MaxLFQ.Unique is only calculated using peptides specific to the given gene.

**Reply by hguturu** [reply]
`Genes.MaxLFQ.Unique` sounds like what I want, but I noticed, this column is blank for many entries. And the blank rate is higher if using `--relaxed-prot-inf --reannotate --pg-level 1`. Does blank mean that peptide in results.tsv is not proteotypic? Why does this blank rate increase if using the different flags?

**Reply by vdemichev** [reply]
Exactly. It's blank for Genes that don't have uniquely matched precursors.

**Reply by vdemichev** [reply]
--reannotate makes proteotypic annotation proper, as opposed to whatever is in the spectral library, which might not be correct, depending on the library. E.g. if the lbirary already contains protein groups and no Proteotypic column, DIA-NN will assume the proteins listed in protein groups are the only ones a precursor is matching to.

**Reply by hguturu** [reply]
Is there a way to get the proteotypic concept to apply at the protein level (e.g. if using `--pg-level 1` as opposed to the default `--pg-level 2` for genes)? I am interested in getting the Unique protein group intensity.

**Reply by bluemooninvestor** [reply]
@hguturu If it is a protein group, then it implies presence of shared peptides. Unique peptides cannot be assigned to groups by definition of groups.

**Reply by vdemichev** [reply]
> Is there a way to get the proteotypic concept to apply at the protein level (e.g. if using --pg-level 1 as opposed to the default --pg-level 2 for genes)? I am interested in getting the Unique protein group intensity.

Yes, exactly by setting the protein inference to isoforms.
